.golangci.yml

```diff
@@ -4,6 +4,7 @@ run:
     - test/i18n
     - pkg/test
   skip-files:
+    - pkg/api/agentPoolOnlyApi/v20180331/apiloader_test.go
```

Comment:
Versions of `golangci-lint` since v1.23.7 have choked on a legal struct literal in this file. I tried many refactorings, then gave up and added it to the "ignored by linter" list.